exporter/importables.go

```diff
@@ -2583,6 +2587,43 @@ var resourcesMap map[string]importable = map[string]importable{
 		Service:        "uc-grants",
 		// TODO: Should we try to make name unique?
 		// TODO: do we need to emit principals? Maybe only on account level? See comment for the owner...
+		Import: func(ic *importContext, r *resource) error {
+			if ic.meUserName == "" {
+				return nil
+			}
+			// TODO: discuss if we need to give ALL_PRIVILEGES vs. giving only specific CREATE permissions, ...
+			// https://docs.databricks.com/en/data-governance/unity-catalog/manage-privileges/privileges.html#privilege-types-by-securable-object-in-unity-catalog
+			var newPrivileges []string
+			if r.Data.Get("metastore").(string) != "" {
+				newPrivileges = append(newPrivileges, metastorePrivilegesToAdd...)
+			} else if r.Data.Get("catalog").(string) != "" || r.Data.Get("schema").(string) != "" ||
```

Comment:
volume & share don't have any child object, so don't need additional privileges granted

Reply:
share yes - agree. But for Volume, should I give `VOLUME_WRITE` if I'm not the owner of the volume and want to write a `databricks_file` ?

Reply:
We emit only Volume files explicitly referenced in jobs/clusters/libraries, not everything.